utils/windows-utils.ts

```diff
@@ -11,4 +14,77 @@ export function validatePackageID(id: string) {
   else {
     throw (`Package ID should be atleast 3 characters long: ${id}`)
   }
+}
+
+const DISALLOWED_WINDOWS_PACKAGE_CHARS_REGEX = /[^a-zA-Z0-9_]/g;
```

Comment:
We also allow period. Is this accounted for?

Reply:
Yep, its not in the list of disallowed chars. For a check i just tried it myself too and a . makes it through this check.